fury/data/tests/test_fetcher.py

```diff
@@ -51,20 +53,29 @@ def tests_fetch_gltf():
 
 
 def test_list_gltf_sample_models():
-    gltf_path = pjoin(fury_home, 'glTF')
-    list_json = pjoin(gltf_path, 'list.json')
-    if not os.path.exists(list_json):
-        json_data = urlopen(f'{GLTF_DATA_URL}').read()
-        with open(list_json, 'wb') as f:
-            f.write(json_data)
-
-    with open(list_json, 'r') as r:
-        data = json.load(r)
-    model_names = [model['name'] for model in data if model['size'] == 0]
     fetch_names = list_gltf_sample_models()
-    npt.assert_equal(len(fetch_names), len(model_names))
-    npt.assert_array_equal(model_names, fetch_names)
-
     default_list = ['BoxTextured', 'Duck', 'CesiumMilkTruck', 'CesiumMan']
     result = [model in fetch_names for model in default_list]
     npt.assert_equal(result, [True, True, True, True])
+
+
+def test_read_viz_gltf():
+    gltf_dir = pjoin(fury_home, 'glTF')
+    filenames, path = fetch_gltf('Box', 'glTF-Binary')
+    filename = read_viz_gltf('Box', 'glTF-Binary')
+    npt.assert_equal(filename, pjoin(path, filenames[0]))
+
+    npt.assert_raises(ValueError, read_viz_gltf, 'FURY', 'glTF')
+
+    box_gltf = pjoin(gltf_dir, 'Box')
+    for path in os.listdir(box_gltf):
+        mode = pjoin(box_gltf, path)
+        for file in os.listdir(mode):
+            os.remove(pjoin(mode, file))
+        os.rmdir(mode)
+    npt.assert_raises(ValueError, read_viz_gltf, 'Box')
+
+    filenames, path = fetch_gltf('Box')
+    out_path = read_viz_gltf('Box').split('/')
```

Comment:
same here. you can use something like `os.sep` instead of `/`

Reply:
Yea, I think this needs to be changed.